css/style.css

```
@@ -0,0 +1,106 @@
+/***************************/
+/* HEADER */
+/**************************/
+.header {
+  display: flex;
+  padding-left: 18rem;
+  padding-right: 18rem;
+  position: absolute;
+  gap: 2.2rem;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
```

This is the only place where we need to set height.
```suggestion
  height: 12rem;
  width: 100%;
}
```